src/js/mdjs.js

```diff
@@ -142,4 +142,4 @@ counties.forEach((name) => {
 //       </div>
 //       <div id="${countyName.toLowerCase()}-dist-row-container" class="dist-row-container"></div>
 // `
-// console.log(counties.sort((a, b) => a.localeCompare(b)).map(name => html(name)).join(""))
\ No newline at end of file
+// console.log(counties.sort((a, b) => a.localeCompare(b)).map(name => html(name)).join(""))
```

Comment:
@apothuku Should we create a scripts folder at the root of the repo and put this and the `raw_data.js` code into it? That way we just have the files already instead of having to comment in/out every time we want to do something.

Reply:
Agreed - created #95